src/main/java/cmput402/tdd/Recipe.java

```diff
@@ -1,4 +1,75 @@
 package cmput402.tdd;
+import org.apache.commons.collections4.map.LinkedMap;
+
+import java.util.Map;
 
 public class Recipe {
-}
+
+    private String name = "";
+    public LinkedMap<Item, Integer> items = new LinkedMap<Item, Integer>();
+
+    public Recipe(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name){
+        this.name = name;
+    }
+
+    public Boolean add(Item item, Integer quantity){
```

Comment:
I think we are moving to throwing an exception if we fail validation checks rather than returning a boolean

Reply:
Changed booleans to now throw exceptions instead